scipy/stats/_morestats.py

```diff
@@ -900,10 +900,12 @@ def boxcox_llf(lmb, data):
 
     .. math::
 
-        llf = (\lambda - 1) \sum_i(\log(x_i)) -
-              N/2 \log(\sum_i (y_i - \bar{y})^2 / N),
+        llf = (\lambda - 1) \sum_i^N(\log(x_i)) -
+              \frac{N}{2} \log(\sum_i^N (y_i - \bar{y})^2 / N),
 
     where ``y`` is the Box-Cox transformed input data ``x``.
+    This corresponds to the profile log- likelihood of the original data ``x`` with
```

Comment:
I'm thinking that the intersection of readers who are interested, don't know this, and understand "profile" without looking at another reference may be small. I'd either leave it out or be a bit more explicit.

Reply:
I think it's useful to keep this information. I had to look up profile likelihood because it had been a while since I'd seen the term. I think it's helpful to know what term to look up.

Reply:
Based on https://github.com/scipy/scipy/pull/22766#discussion_r2023059863, I see now that the intent was to disambiguate this from the full log-likelihood (including scale and location). That makes it seem more useful to me to those who are interested.